src/app/pages/storage/disks/disk-form/disk-form.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import { createComponentFactory, mockProvider, Spectator } from '@ngneat/spectator/jest';
+import { mockCall, mockWebsocket } from 'app/core/testing/utils/mock-websocket.utils';
+import { DiskBus } from 'app/enums/disk-bus.enum';
+import { DiskPowerLevel } from 'app/enums/disk-power-level.enum';
+import { DiskStandby } from 'app/enums/disk-standby.enum';
+import { DiskType } from 'app/enums/disk-type.enum';
+import { Disk } from 'app/interfaces/storage.interface';
+import { IxCheckboxHarness } from 'app/modules/ix-forms/components/ix-checkbox/ix-checkbox.harness';
+import { IxInputHarness } from 'app/modules/ix-forms/components/ix-input/ix-input.harness';
+import { IxFormsModule } from 'app/modules/ix-forms/ix-forms.module';
+import { IxFormHarness } from 'app/modules/ix-forms/testing/ix-form.harness';
+import { DialogService, WebSocketService } from 'app/services';
+import { IxSlideInService } from 'app/services/ix-slide-in.service';
+import { DiskFormComponent } from './disk-form.component';
+
+describe('DiskFormComponent', () => {
+  let spectator: Spectator<DiskFormComponent>;
+  let loader: HarnessLoader;
+  let form: IxFormHarness;
+  const dataDisk: Disk = {
+    name: 'sdc',
+    serial: 'VB9fbb6dfe-9cf26570',
+    advpowermgmt: DiskPowerLevel.Level127,
+    critical: '5',
+    description: 'Some disk description',
+    difference: '5',
+    informational: '5',
+    hddstandby: DiskStandby.Minutes10,
+    passwd: '123',
+    smartoptions: 'smart options',
+    togglesmart: false,
+    bus: DiskBus.Spi,
+    devname: 'sdc',
+    enclosure: {} as any,
+    expiretime: '',
+    identifier: '{serial}VB9fbb6dfe-9cf26570',
+    model: 'VBOX_HARDDISK',
+    number: 3,
+    pool: 'pool1',
+    rotationrate: 1,
+    size: 5368709120,
+    subsystem: 'scsi',
+    transfermode: 'Auto',
+    type: DiskType.Hdd,
+    zfs_guid: '12615153293172479823',
+    duplicate_serial: [''],
+    multipath_member: '',
+    multipath_name: '',
+  };
+  const createComponent = createComponentFactory({
+    component: DiskFormComponent,
+    imports: [
+      IxFormsModule,
+      ReactiveFormsModule,
+    ],
+    providers: [
+      mockProvider(IxSlideInService),
+      mockProvider(DialogService),
+      mockWebsocket([
+        mockCall('disk.update', dataDisk),
+      ]),
+    ],
+  });
+
+  beforeEach(async () => {
+    spectator = createComponent();
+    loader = TestbedHarnessEnvironment.loader(spectator.fixture);
+    form = await loader.getHarness(IxFormHarness);
+  });
+
+  it('the field \'SED Password\' should be disabled if the checkbox \'Clear SED Password\' value is true', async () => {
```

Comment:
`it` is part of description, so `it disables SED Password when Clear SED Password is checked`